src/app/app.routes.ts

```diff
@@ -1,7 +1,12 @@
 import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './home/home.component';
-
+import { WorkshopsComponent } from './workshops/workshops.component';
+import { WorkshopDetailsComponent } from './workshop-details/workshop-details.component'
+import { ContactUsComponent } from './contact-us/contact-us.component'
 
 export const ROUTES: Routes = [
-  { path: '', component: HomeComponent }
+  { path: '', component: HomeComponent },
+  { path: 'workshops', component: WorkshopsComponent },
+  { path: 'photography-workshop-details/:title/:id', component: WorkshopDetailsComponent },
```

Comment:
Nice! Can we also append location here? just for SEO purposes. And all spaces need to be replaced with '-'. So "ships at the harbor" workshop in "seattle Washington" becomes "ships-at-the-harbor-seattle-Washington-workshop"